src/nvim/generators/gen_options.lua

```diff
@@ -41,6 +42,12 @@ local list_flags = {
   flagscomma = 'P_COMMA|P_FLAGLIST',
 }
 
+--- @param s string
+--- @return string
+local title_case = function(s)
+  return s:sub(1, 1):upper() .. s:sub(2):lower()
```

Comment:
Isn't all options lowercase anyway so only the `:upper()` part is needed here? Also, if we ever were to add cased option names (which we will not, but if we would), it would be fine to hypothetically preserve that.